### PR TITLE
LPD-42552 Add check permission for fix a test that is failing

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-42552

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-asset-categories-navigation-web/src/main/java/com/liferay/commerce/product/asset/categories/navigation/web/internal/display/context/CPAssetCategoriesNavigationDisplayContext.java [hunk 1]
```diff
@@ -141,13 +146,21 @@ public AssetVocabulary getAssetVocabulary() throws PortalException {
...
-		return _assetVocabulary;
+		if ((_assetVocabulary != null) &&
+			AssetVocabularyPermission.contains(
+				_themeDisplay.getPermissionChecker(), _assetVocabulary,
+				ActionKeys.VIEW)) {
+
+			return _assetVocabulary;
+		}
+
+		return null;
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-asset-categories-navigation-web/src/main/java/com/liferay/commerce/product/asset/categories/navigation/web/internal/display/context/CPAssetCategoriesNavigationDisplayContext.java:163]: You need to cover the scenario in which vocabulary is null

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@monaco-fabio please check my comment. Thank you!

**Comment by monaco-fabio** [comment]
ci:stop

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:commerce-functional

**Comment by monaco-fabio** [comment]
ci:report:91589824

**Review by alessiorendina — APPROVED** [review]

**Comment by alessiorendina** [comment]
@monaco-fabio confirmed that failures are not related to the PR, thank you!

**Comment by alessiorendina** [comment]
ci:forward:force

**Pull request closed without merging**